github/github.go

```diff
@@ -135,6 +135,9 @@ const (
 
 	// https://developer.github.com/changes/2019-10-03-multi-line-comments/
 	mediaTypeMultiLineCommentsPreview = "application/vnd.github.comfort-fade-preview+json"
+
+	// https://developer.github.com/v3/previews/#new-oauth-applications-api-endpoints
```

Comment:
I'm sorry I wasn't clear before.  The URLs here are for the blog posts, so this one should be:
```
// https://developer.github.com/changes/2019-11-05-deprecated-passwords-and-authorizations-api
```

Reply:
Got it